app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -73,7 +73,8 @@ $red: #E61E32;
     border-top: 1px solid govuk-colour("mid-grey", $legacy: "grey-2");
     content: "";
     display: block;
-    margin: 0 govuk-spacing(3) govuk-spacing(6);
+    margin: 0 govuk-spacing(3);
+    @include govuk-responsive-margin(6, "bottom");
```

Comment:
Interesting, so what does the compile to, something like:

```
margin: 0 15px;
margin-bottom: 30px
@media only screen and (min-width: $tablet) {
  margin-bottom: 20px
}
```

Line 76 already would already apply a margin-bottom 15px right? so is that just overridden by the mixin at small screen sizes or does it also give a desktop overwrite?

Reply:
Yes, the `margin-bottom` will override  any margin applied by `margin`. Line 76 actually means the margin-bottom will be set to 0 and the left value will be set to 15px.

I guess a clearer way of writing it might be to split each one out separately, e.g:

```
margin-right: 15px
margin-left: 15px
@include govuk-responsive-margin(6, "bottom");
```

I'll amend to make it clearer 👍